src/sql-parser/tests/testdata/ddl

```diff
@@ -728,11 +728,11 @@ CREATE INDEX foo ON myschema.bar (a, b)
 CreateIndex(CreateIndexStatement { name: Some(Ident("foo")), in_cluster: None, on_name: Name(UnresolvedObjectName([Ident("myschema"), Ident("bar")])), key_parts: Some([Identifier([Ident("a")]), Identifier([Ident("b")])]), with_options: [], if_not_exists: false })
 
 parse-statement
-CREATE INDEX foo ON myschema.bar (a, b) WITH (baz = 'raz')
+CREATE INDEX foo ON myschema.bar (a, b) WITH LOGICAL COMPACTION WINDOW = 0
```

Comment:
nit: I think we should keep the parens here. Probably best to follow PostgreSQL a _bit_ closer here (even though multi-keyword arguments are our invention), in case we ever want to support `CREATE INDEX ... WHERE`.